### DISCUSSION TITLE
Remove Backend

### DISCUSSION THREAD

**Discussion by deathbybandaid** [arabcoders/watchstate#313] [General] [answered]
Simple question, is there a way to remove backends? I can't seem to find anything in the docs for this.

**Comment by arabcoders** [comment, marked as answer]
> Simple question, is there a way to remove backends? I can't seem to find anything in the docs for this.

Hi, indeed i never considered the possibility of removing the backend. It should be relatively easy to add. 

I am busy at the moment with something else. However, it will be added. 

In the meantime you could just remove the db and the entry from `servers.yaml` file. 

Thank you for your suggestion. #315
